### ISSUE TITLE
Add Drupal to Alpaca Travis Setup

### ISSUE THREAD

**Issue by dannylamb** [Islandora-CLAW/CLAW#455]
We need Drupal installed in the Alpaca test environment before we can do end-to-end integration testing.  Add the relevant parts of the [travis setup for the islandora modules](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/.scripts/travis_setup.sh) to Alpaca.

**Comment by ruebot** [comment]
@dannylamb questions about the priority and need of this as I am working through https://github.com/Islandora-CLAW/Alpaca/pull/23:
  - What's the priority on this?
  - Is this 100%, or can we achieve the end-to-end testing another way

Asking because of the constraints of TravisCI. We are bound to an Ubuntu 12.04 machine, and that machine is either a "Java" or "PHP" machine. So, if it is a Java machine, which we are using now, we need to setup a full PHP 7 environment on it, along with MySQL and PEAR to get everything going. Or, we go the other way, and get Java environment setup on a PHP machine. So, before I go any further on this, let me know what you think.

Current status of the build with attempting a full Drupal setup can be found here: https://travis-ci.org/Islandora-CLAW/Alpaca/jobs/185176564#L171

**Comment by dannylamb** [comment]
For posterity, there was an IRC conversation about this, and we decided to move forward with putting Java onto a PHP box, since LAMP is trickier to install via travis than just Java.

**Comment by ruebot** [comment]
http://irclogs.islandora.ca/2016-12-19.html

**Comment by dannylamb** [comment]
@ruebot Why are we bound to 12.04?  I don't like the thought of being two LTS's behind if it can be avoided.

**Comment by ruebot** [comment]
https://docs.travis-ci.com/user/ci-environment/ -- Honestly not sure how to force it to use 14.04, which is supposedly available.

**Comment by dannylamb** [comment]
hunh, apparantly like this:
```
dist: trusty
sudo: required
```
totally out of scope for this, but good to know.

**Comment by ruebot** [comment]
Hah! That totally makes sense! I'm happy to toss another commit on here if you'd like just to see if it works, and if it does, we can create a ticket to update other builds. Seems like that would be a good newbie ticket too.

**Comment by ruebot** [comment]
@dannylamb added `trusty` in the latest code review commit.

**Comment by dannylamb** [comment]
you didn't have to.  but as long as there's no explosions, i don't mind ;)
